DEBUG 2020-02-04 23:16:35.391 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-04 23:16:35.404 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-04 23:16:35.404 UTC: 	 Executing command: 'preparelocal'
DEBUG 2020-02-04 23:16:35.404 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-02-04 23:16:45.690 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-04 23:16:45.704 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-04 23:16:45.705 UTC: 	 Executing command: 'preparelocal'
DEBUG 2020-02-04 23:16:45.705 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-02-05 00:26:03.468 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-05 00:26:03.483 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-05 00:26:03.483 UTC: 	 Executing command: 'status'
DEBUG 2020-02-05 00:26:03.484 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-02-05 00:26:03.485 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon
ERROR 2020-02-05 00:26:03.485 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 599, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 461, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon
DEBUG 2020-02-05 00:34:17.940 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-05 00:34:17.954 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-05 00:34:17.954 UTC: 	 Executing command: 'submit'
DEBUG 2020-02-05 00:34:17.962 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-02-05 00:34:17.968 UTC: 	 Checking credentials
DEBUG 2020-02-05 00:34:17.968 UTC: 	 Found proxy file /tmp/x509up_u10060
DEBUG 2020-02-05 00:34:17.968 UTC: 	 Getting proxy life time left
DEBUG 2020-02-05 00:34:18.072 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 604530

 error: 
 retcode : 0
DEBUG 2020-02-05 00:34:18.176 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 604530

 error: 
 retcode : 0
DEBUG 2020-02-05 00:34:18.176 UTC: 	 Proxy valid for 167:55:30 hours
DEBUG 2020-02-05 00:34:18.177 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2020-02-05 00:34:19.544 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-05 00:34:19.558 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-05 00:34:19.558 UTC: 	 Executing command: 'submit'
DEBUG 2020-02-05 00:34:19.558 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-02-05 00:34:19.564 UTC: 	 Checking credentials
DEBUG 2020-02-05 00:34:19.564 UTC: 	 Found proxy file /tmp/x509up_u10060
DEBUG 2020-02-05 00:34:19.564 UTC: 	 Getting proxy life time left
DEBUG 2020-02-05 00:34:19.668 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 604529

 error: 
 retcode : 0
DEBUG 2020-02-05 00:34:19.772 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 604529

 error: 
 retcode : 0
DEBUG 2020-02-05 00:34:19.773 UTC: 	 Proxy valid for 167:55:29 hours
DEBUG 2020-02-05 00:34:19.773 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2020-02-05 00:35:59.708 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-05 00:35:59.722 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-05 00:35:59.722 UTC: 	 Executing command: 'kill'
DEBUG 2020-02-05 00:35:59.723 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-02-05 00:35:59.723 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_noMt2W/crab_sample2
ERROR 2020-02-05 00:35:59.724 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 599, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 461, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_noMt2W/crab_sample2
DEBUG 2020-02-09 21:30:21.466 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.201-1.el7.elrepo.x86_64 #1 SMP Tue Nov 12 14:34:24 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-09 21:30:21.531 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-09 21:30:21.531 UTC: 	 Executing command: 'status'
DEBUG 2020-02-09 21:30:21.532 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-02-09 21:30:21.533 UTC: 	 Error: ModuleCommon_noMt2W/crab_WPlusJets_sample17 is not a valid CRAB project directory.
ERROR 2020-02-09 21:30:21.534 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_noMt2W/crab_WPlusJets_sample17 is not a valid CRAB project directory.
DEBUG 2020-02-11 05:12:03.371 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-11 05:12:03.432 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-11 05:12:03.433 UTC: 	 Executing command: 'status'
DEBUG 2020-02-11 05:12:03.433 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-02-11 05:12:03.434 UTC: 	 Error: ModuleCommon_noMt2W/crab_ttbarPlusJets_sample2 is not a valid CRAB project directory.
ERROR 2020-02-11 05:12:03.434 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_noMt2W/crab_ttbarPlusJets_sample2 is not a valid CRAB project directory.
DEBUG 2020-02-26 07:57:38.606 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-02-26 07:57:38.623 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-02-26 07:57:38.624 UTC: 	 Executing command: 'status'
DEBUG 2020-02-26 07:57:38.624 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-02-26 07:57:38.625 UTC: 	 Error: keepanddropv1/crab_ttbarPlusJets_sample0/ is not a valid CRAB project directory.
ERROR 2020-02-26 07:57:38.625 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: keepanddropv1/crab_ttbarPlusJets_sample0/ is not a valid CRAB project directory.
DEBUG 2020-03-05 23:48:12.884 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-05 23:48:12.901 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-05 23:48:12.901 UTC: 	 Executing command: 'status'
DEBUG 2020-03-05 23:48:12.901 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-05 23:48:12.903 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHistv2/crab_ttbarPlusJets_sample0
ERROR 2020-03-05 23:48:12.903 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 599, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 461, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHistv2/crab_ttbarPlusJets_sample0
DEBUG 2020-03-05 23:48:29.284 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-05 23:48:29.298 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-05 23:48:29.298 UTC: 	 Executing command: 'status'
DEBUG 2020-03-05 23:48:29.299 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-05 23:48:29.300 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHistv2/crab_ttbarPlusJets_sample0
ERROR 2020-03-05 23:48:29.300 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 599, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 461, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHistv2/crab_ttbarPlusJets_sample0
DEBUG 2020-03-06 00:33:41.240 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-06 00:33:41.253 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-06 00:33:41.254 UTC: 	 Executing command: 'submit'
DEBUG 2020-03-06 00:33:41.264 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-03-06 00:33:41.270 UTC: 	 Checking credentials
DEBUG 2020-03-06 00:33:41.270 UTC: 	 Found proxy file /tmp/x509up_u10060
DEBUG 2020-03-06 00:33:41.270 UTC: 	 Getting proxy life time left
DEBUG 2020-03-06 00:33:41.374 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 43195

 error: 
 retcode : 0
DEBUG 2020-03-06 00:33:41.478 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 43195

 error: 
 retcode : 0
DEBUG 2020-03-06 00:33:41.478 UTC: 	 Proxy valid for 11:59:55 hours
DEBUG 2020-03-06 00:33:41.479 UTC: 	 Will use CRAB Analysis plugin (i.e. will run an analysis job type).
DEBUG 2020-03-06 02:58:47.427 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-06 02:58:47.443 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-06 02:58:47.443 UTC: 	 Executing command: 'status'
DEBUG 2020-03-06 02:58:47.443 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-06 02:58:47.444 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_ttbarPlusJets_sample1
ERROR 2020-03-06 02:58:47.444 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 599, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 461, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_ttbarPlusJets_sample1
DEBUG 2020-03-06 03:19:13.179 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-06 03:19:13.195 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-06 03:19:13.196 UTC: 	 Executing command: 'status'
DEBUG 2020-03-06 03:19:13.196 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-06 03:19:13.197 UTC: 	 Error: ModuleCommon_withKFactors/crab_WZ_sample6 is not a valid CRAB project directory.
ERROR 2020-03-06 03:19:13.197 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_withKFactors/crab_WZ_sample6 is not a valid CRAB project directory.
DEBUG 2020-03-06 03:35:10.878 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-06 03:35:10.895 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-06 03:35:10.896 UTC: 	 Executing command: 'status'
DEBUG 2020-03-06 03:35:10.896 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-06 03:35:10.897 UTC: 	 Error: ModuleCommon_withKFactors/crab_QCD_sample15 is not a valid CRAB project directory.
ERROR 2020-03-06 03:35:10.897 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_withKFactors/crab_QCD_sample15 is not a valid CRAB project directory.
DEBUG 2020-03-06 23:12:18.486 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-06 23:12:18.502 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-06 23:12:18.502 UTC: 	 Executing command: 'status'
DEBUG 2020-03-06 23:12:18.503 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-03-08 04:24:11.044 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-08 04:24:11.059 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-08 04:24:11.060 UTC: 	 Executing command: 'status'
DEBUG 2020-03-08 04:24:11.060 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-03-23 17:02:01.810 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.205-1.el7.elrepo.x86_64 #1 SMP Fri Nov 29 10:10:01 EST 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-03-23 17:02:01.824 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-03-23 17:02:01.825 UTC: 	 Executing command: 'status'
DEBUG 2020-03-23 17:02:01.825 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-03-23 17:02:01.826 UTC: 	 Error: getBTagHist/crab_WPlusJets_sample17 is not a valid CRAB project directory.
ERROR 2020-03-23 17:02:01.826 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist/crab_WPlusJets_sample17 is not a valid CRAB project directory.
DEBUG 2020-04-02 02:16:59.202 UTC: 	 Running on: Linux login02.hep.wisc.edu 4.4.190-1.el7.elrepo.x86_64 #1 SMP Sun Aug 25 07:32:44 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-04-02 02:16:59.222 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-04-02 02:16:59.223 UTC: 	 Executing command: 'checkusername'
DEBUG 2020-04-02 02:16:59.224 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-04-02 02:16:59.233 UTC: 	 Checking credentials
DEBUG 2020-04-02 02:16:59.234 UTC: 	 Found proxy file /tmp/x509up_u10060
DEBUG 2020-04-02 02:16:59.234 UTC: 	 Getting proxy life time left
DEBUG 2020-04-02 02:16:59.339 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 43134

 error: 
 retcode : 0
DEBUG 2020-04-02 02:16:59.444 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 
 error: 
 retcode : 1
DEBUG 2020-04-02 02:16:59.444 UTC: 	 Proxy lifetime 43134 secs is different from voms extension lifetime 0 secs for proxy: /tmp/x509up_u10060
DEBUG 2020-04-02 02:16:59.444 UTC: 	 Proxy valid for 00:00:00 hours
DEBUG 2020-04-02 02:16:59.444 UTC: 	 Creating new proxy for 24:00 hours with VO group '' and VO role 'NULL'.
DEBUG 2020-04-02 02:17:03.654 UTC: 	 Executing : 
 command : voms-proxy-init -voms cms:/cms -valid 24:00 -rfc
 output : None
 error: None
 retcode : 0
DEBUG 2020-04-02 02:17:03.759 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 86400

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:03.864 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 86400

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:03.969 UTC: 	 Executing : 
 command : env X509_USER_PROXY=/tmp/x509up_u10060 voms-proxy-info -fqan
 output : /cms/Role=NULL/Capability=NULL
/cms/uscms/Role=NULL/Capability=NULL

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:03.970 UTC: 	 Proxy created.
DEBUG 2020-04-02 02:17:04.082 UTC: 	 Executing : 
 command : openssl x509 -in ~/.globus/usercert.pem -subject -noout
 output : subject= /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=vshang/CN=836015/CN=Victor Shang

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:04.082 UTC: 	 Checking credentials
DEBUG 2020-04-02 02:17:04.082 UTC: 	 Found proxy file /tmp/x509up_u10060
DEBUG 2020-04-02 02:17:04.082 UTC: 	 Getting proxy life time left
DEBUG 2020-04-02 02:17:04.187 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -timeleft
 output : 86399

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:04.292 UTC: 	 Executing : 
 command : voms-proxy-info -file /tmp/x509up_u10060 -actimeleft
 output : 86399

 error: 
 retcode : 0
DEBUG 2020-04-02 02:17:04.293 UTC: 	 Proxy valid for 23:59:59 hours
DEBUG 2020-04-02 02:17:04.293 UTC: 	 Command use: checkusername
DEBUG 2020-04-02 02:17:04.293 UTC: 	 Options use: []
DEBUG 2020-04-02 02:17:04.293 UTC: 	 Instance is None
DEBUG 2020-04-02 02:17:04.293 UTC: 	 Server base url is None
INFO 2020-04-02 02:17:04.293 UTC: 	 Retrieving DN from proxy...
INFO 2020-04-02 02:17:04.535 UTC: 	 DN is: /DC=ch/DC=cern/OU=Organic Units/OU=Users/CN=vshang/CN=836015/CN=Victor Shang
INFO 2020-04-02 02:17:04.535 UTC: 	 Retrieving username for this DN...
DEBUG 2020-04-02 02:17:04.535 UTC: 	 Retrieving username from SiteDB...
DEBUG 2020-04-02 02:17:05.878 UTC: 	 Error UsernameException: Failed to retrieve username from SiteDB. Your DN does not seem to be registered in SiteDB.
Details follow:
  Executed command: curl -sS --capath /etc/grid-security/certificates --cert /tmp/x509up_u10060 --key /tmp/x509up_u10060 'https://cmsweb.cern.ch/sitedb/data/prod/whoami'
    Stdout:
      <!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.1//EN" "http://www.w3.org/TR/xhtml11/DTD/xhtml11.dtd">
      <html xmlns="http://www.w3.org/1999/xhtml" xml:lang="en">
       <head>
       <title>CMSWEB Error: No such site</title>
       <meta http-equiv="Content-Type" content="text/html;charset=utf-8" />
       <link rel="shortcut icon" href="/favicon.ico" type="image/x-icon" />
       <link rel="stylesheet" type="text/css" media="screen" href="/css/cmsweb.css" />
      </head>
      <body>
       <div id="main">
        <div id="top">
         <div class="boxTitle"><img src="/img/title.gif" alt="CMSWEB (title)" /></div>
        </div>
        <div id="middle">
         <div class="boxLinkContainer">
          <h2>No such site</h2>
          <p>The site you requested does not exist. Could we interest you in <a href='/'>anything else</a>?</p>
         </div>
        </div>
        <div id="bottom">
         <div class="boxFooter">&nbsp;</div>
        </div>
       </div>
      </body>
      </html>
      
    Parsed username: 
Note: Make sure you have the correct certificate mapped in SiteDB (you can check what is the certificate you currently have mapped in SiteDB by searching for your name in https://cmsweb.cern.ch/sitedb/prod/people). For instructions on how to map a certificate in SiteDB, see https://twiki.cern.ch/twiki/bin/viewauth/CMS/SiteDBForCRAB.
DEBUG 2020-04-02 02:17:05.878 UTC: 	 Retrieving username from CRIC...
DEBUG 2020-04-02 02:17:08.660 UTC: 	 Username is: vshang
INFO 2020-04-02 02:17:08.661 UTC: 	 Failed to get username from SiteDB.
 Details are in crab.log file
Will try new Coputing Resource Information Catalog CRIC
INFO 2020-04-02 02:17:08.661 UTC: 	 username is vshang
DEBUG 2020-04-07 15:48:00.218 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-04-07 15:48:00.231 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-04-07 15:48:00.232 UTC: 	 Executing command: 'resubmit'
DEBUG 2020-04-07 15:48:00.232 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-04-07 15:48:00.233 UTC: 	 Error: ModuleCommon_withPUandBTagWeights/crab_WZ_sampl5 is not a valid CRAB project directory.
ERROR 2020-04-07 15:48:00.233 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/resubmit.py", line 28, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_withPUandBTagWeights/crab_WZ_sampl5 is not a valid CRAB project directory.
DEBUG 2020-04-15 19:53:14.532 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-04-15 19:53:14.549 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-04-15 19:53:14.549 UTC: 	 Executing command: 'status'
DEBUG 2020-04-15 19:53:14.550 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-04-15 19:53:14.551 UTC: 	 Error: ModuleCommon_withPUandBTagWeightsv2/crab_SingleMuon_Run2016_sample8 is not a valid CRAB project directory.
ERROR 2020-04-15 19:53:14.551 UTC: 	 Caught exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 160, in <module>
    client()
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/bin/crab", line 145, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/status.py", line 40, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 277, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/crab3/slc6_amd64_gcc493/cms/crabclient/3.3.2001/lib/python2.7/site-packages/CRABClient/Commands/SubCommand.py", line 595, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_withPUandBTagWeightsv2/crab_SingleMuon_Run2016_sample8 is not a valid CRAB project directory.
DEBUG 2020-05-24 06:44:54.053 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-24 06:44:54.054 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-24 06:44:54.068 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-24 06:44:54.069 UTC: 	 Executing command: 'status'
DEBUG 2020-05-24 06:44:54.069 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-24 06:44:54.070 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-24 06:44:54.070 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-24 06:44:54.114 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-24 06:45:13.609 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-24 06:45:13.609 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-24 06:45:13.651 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-24 06:45:13.651 UTC: 	 Executing command: 'status'
DEBUG 2020-05-24 06:45:13.651 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-24 06:45:13.652 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-24 06:45:13.652 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-24 06:45:13.654 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-24 06:46:37.064 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-24 06:46:37.064 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-24 06:46:37.078 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-24 06:46:37.078 UTC: 	 Executing command: 'status'
DEBUG 2020-05-24 06:46:37.079 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-24 06:46:37.080 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-24 06:46:37.080 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-24 06:46:37.083 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-24 06:47:11.989 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-24 06:47:11.989 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-24 06:47:12.003 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-24 06:47:12.004 UTC: 	 Executing command: 'resubmit'
DEBUG 2020-05-24 06:47:12.004 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-24 06:47:12.005 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-24 06:47:12.005 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/resubmit.py", line 28, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-24 06:47:12.007 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-24 23:47:06.667 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-24 23:47:06.667 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-24 23:47:06.680 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-24 23:47:06.681 UTC: 	 Executing command: 'status'
DEBUG 2020-05-24 23:47:06.681 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-24 23:47:06.683 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-24 23:47:06.683 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-24 23:47:06.701 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-27 00:11:11.745 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-27 00:11:11.746 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-27 00:11:11.764 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-27 00:11:11.764 UTC: 	 Executing command: 'status'
DEBUG 2020-05-27 00:11:11.765 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-27 00:11:11.768 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
ERROR 2020-05-27 00:11:11.768 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/test/crab_MET_Run2016_sample0
INFO 2020-05-27 00:11:11.775 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-27 01:05:37.785 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-27 01:05:37.785 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-27 01:05:37.803 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-27 01:05:37.804 UTC: 	 Executing command: 'status'
DEBUG 2020-05-27 01:05:37.804 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-27 01:05:37.805 UTC: 	 Error: ModuleCommon_05242020/crab_WZ_sample is not a valid CRAB project directory.
ERROR 2020-05-27 01:05:37.805 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_05242020/crab_WZ_sample is not a valid CRAB project directory.
INFO 2020-05-27 01:05:37.809 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-05-27 01:07:53.461 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-05-27 01:07:53.462 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-05-27 01:07:53.479 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-05-27 01:07:53.479 UTC: 	 Executing command: 'status'
DEBUG 2020-05-27 01:07:53.480 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-05-27 01:07:53.481 UTC: 	 Error: ModuleCommon_05242020/crab_WZ_sample12 is not a valid CRAB project directory.
ERROR 2020-05-27 01:07:53.481 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_05242020/crab_WZ_sample12 is not a valid CRAB project directory.
INFO 2020-05-27 01:07:53.484 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-01 16:54:54.965 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-01 16:54:54.966 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-01 16:54:55.001 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-01 16:54:55.001 UTC: 	 Executing command: 'resubmit'
DEBUG 2020-06-01 16:54:55.001 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
DEBUG 2020-06-03 17:03:52.588 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-03 17:03:52.588 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-03 17:03:52.608 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-03 17:03:52.609 UTC: 	 Executing command: 'status'
DEBUG 2020-06-03 17:03:52.610 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-03 17:03:52.612 UTC: 	 Error: ModuleCommon_06022020/crab_WW_sample3 is not a valid CRAB project directory.
ERROR 2020-06-03 17:03:52.612 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/3.3.2005-bcolbf2/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_06022020/crab_WW_sample3 is not a valid CRAB project directory.
INFO 2020-06-03 17:03:52.634 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-11 08:34:22.234 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-11 08:34:22.234 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-11 08:34:22.252 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-11 08:34:22.252 UTC: 	 Executing command: 'status'
DEBUG 2020-06-11 08:34:22.252 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-11 08:34:22.254 UTC: 	 Error: getBTagHist_06082020/crab_ZTo2L_Run2017_sampleqq is not a valid CRAB project directory.
ERROR 2020-06-11 08:34:22.254 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_06082020/crab_ZTo2L_Run2017_sampleqq is not a valid CRAB project directory.
INFO 2020-06-11 08:34:22.279 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-13 04:31:43.589 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-13 04:31:43.590 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-13 04:31:43.610 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-13 04:31:43.610 UTC: 	 Executing command: 'status'
DEBUG 2020-06-13 04:31:43.611 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-13 04:31:43.691 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_06122020/crab_ZZ_Run2016_sample2
ERROR 2020-06-13 04:31:43.691 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_06122020/crab_ZZ_Run2016_sample2
INFO 2020-06-13 04:31:43.699 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-13 04:32:22.042 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-13 04:32:22.043 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-13 04:32:22.136 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-13 04:32:22.137 UTC: 	 Executing command: 'kill'
DEBUG 2020-06-13 04:32:22.137 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-13 04:32:22.138 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_06122020/crab_ZZ_Run2016_sample2
ERROR 2020-06-13 04:32:22.138 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/ModuleCommon_06122020/crab_ZZ_Run2016_sample2
INFO 2020-06-13 04:32:22.150 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-14 19:05:26.433 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-14 19:05:26.434 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-14 19:05:26.469 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-14 19:05:26.470 UTC: 	 Executing command: 'status'
DEBUG 2020-06-14 19:05:26.470 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-14 19:05:26.472 UTC: 	 Error: getBTagHist_06082020/crab_WZ_Run2017_sample4 is not a valid CRAB project directory.
ERROR 2020-06-14 19:05:26.473 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_06082020/crab_WZ_Run2017_sample4 is not a valid CRAB project directory.
INFO 2020-06-14 19:05:26.490 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-15 07:53:22.882 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-15 07:53:22.883 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-15 07:53:22.919 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-15 07:53:22.919 UTC: 	 Executing command: 'status'
DEBUG 2020-06-15 07:53:22.920 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-15 07:53:22.922 UTC: 	 Error: getBTagHist_06082020/crab_WW_Run2017_sample3 is not a valid CRAB project directory.
ERROR 2020-06-15 07:53:22.923 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_06082020/crab_WW_Run2017_sample3 is not a valid CRAB project directory.
INFO 2020-06-15 07:53:22.930 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-15 16:30:50.826 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-15 16:30:50.827 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-15 16:30:50.861 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-15 16:30:50.862 UTC: 	 Executing command: 'status'
DEBUG 2020-06-15 16:30:50.862 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-15 16:30:50.863 UTC: 	 Error: getBTagHist_06082020/crab_ZZ_Run2017_sample7 is not a valid CRAB project directory.
ERROR 2020-06-15 16:30:50.863 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_06082020/crab_ZZ_Run2017_sample7 is not a valid CRAB project directory.
INFO 2020-06-15 16:30:50.874 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-16 07:32:35.251 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-16 07:32:35.251 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-16 07:32:35.278 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-16 07:32:35.279 UTC: 	 Executing command: 'kill'
DEBUG 2020-06-16 07:32:35.279 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-16 07:32:35.281 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_06082020/crab_ttbarPlusJets_Run2017_sample0
ERROR 2020-06-16 07:32:35.281 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_06082020/crab_ttbarPlusJets_Run2017_sample0
INFO 2020-06-16 07:32:35.296 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-16 07:34:23.433 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-16 07:34:23.434 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-16 07:34:23.471 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-16 07:34:23.472 UTC: 	 Executing command: 'kill'
DEBUG 2020-06-16 07:34:23.473 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-16 07:34:23.477 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_06082020/crab_ttbarPlusJets_Run2017_sample0
ERROR 2020-06-16 07:34:23.478 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_06082020/crab_ttbarPlusJets_Run2017_sample0
INFO 2020-06-16 07:34:23.494 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-17 20:47:43.659 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-17 20:47:43.659 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-17 20:47:43.681 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-17 20:47:43.682 UTC: 	 Executing command: 'status'
DEBUG 2020-06-17 20:47:43.682 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-17 20:47:43.683 UTC: 	 Error: getBTagHist_06082020/crab_WPlusJets_Run2017_sample13 is not a valid CRAB project directory.
ERROR 2020-06-17 20:47:43.683 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_06082020/crab_WPlusJets_Run2017_sample13 is not a valid CRAB project directory.
INFO 2020-06-17 20:47:43.705 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-06-28 11:14:20.153 UTC: 	 CRAB Client version: 3.3.2005
DEBUG 2020-06-28 11:14:20.154 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-06-28 11:14:20.181 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-06-28 11:14:20.181 UTC: 	 Executing command: 'status'
DEBUG 2020-06-28 11:14:20.182 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-06-28 11:14:20.184 UTC: 	 Error: ModuleCommon_06282020/crab_ZTo2L_Run2017_sample12 is not a valid CRAB project directory.
ERROR 2020-06-28 11:14:20.184 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.2005.01/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_06282020/crab_ZTo2L_Run2017_sample12 is not a valid CRAB project directory.
INFO 2020-06-28 11:14:20.216 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-15 18:01:56.560 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-15 18:01:56.561 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-15 18:01:56.583 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-15 18:01:56.584 UTC: 	 Executing command: 'status'
DEBUG 2020-07-15 18:01:56.585 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-15 18:01:56.587 UTC: 	 Error: ModuleCommon_07142020/crab_WW_Run2018_sample3 is not a valid CRAB project directory.
ERROR 2020-07-15 18:01:56.587 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_07142020/crab_WW_Run2018_sample3 is not a valid CRAB project directory.
INFO 2020-07-15 18:01:56.616 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-15 18:03:10.960 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-15 18:03:10.961 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-15 18:03:10.996 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-15 18:03:10.997 UTC: 	 Executing command: 'status'
DEBUG 2020-07-15 18:03:10.997 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-15 18:03:10.999 UTC: 	 Error: ModuleCommon_07142020/crab_WZ_Run2018_sample5 is not a valid CRAB project directory.
ERROR 2020-07-15 18:03:11.000 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_07142020/crab_WZ_Run2018_sample5 is not a valid CRAB project directory.
INFO 2020-07-15 18:03:11.006 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-15 18:05:11.635 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-15 18:05:11.636 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-15 18:05:11.671 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-15 18:05:11.672 UTC: 	 Executing command: 'status'
DEBUG 2020-07-15 18:05:11.672 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-15 18:05:11.674 UTC: 	 Error: ModuleCommon_07142020/crab_ZZ_Run2018_sample6 is not a valid CRAB project directory.
ERROR 2020-07-15 18:05:11.675 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_07142020/crab_ZZ_Run2018_sample6 is not a valid CRAB project directory.
INFO 2020-07-15 18:05:11.681 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-23 03:13:53.619 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-23 03:13:53.620 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-23 03:13:53.657 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-23 03:13:53.658 UTC: 	 Executing command: 'kill'
DEBUG 2020-07-23 03:13:53.659 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-23 03:13:53.661 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_DeepCSV_07222020/crab_singleTop_Run2016_sample4
ERROR 2020-07-23 03:13:53.662 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_DeepCSV_07222020/crab_singleTop_Run2016_sample4
INFO 2020-07-23 03:13:53.680 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-23 08:19:58.478 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-23 08:19:58.479 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-23 08:19:58.525 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-23 08:19:58.526 UTC: 	 Executing command: 'status'
DEBUG 2020-07-23 08:19:58.526 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-23 08:19:58.529 UTC: 	 Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_DeepCSV_07232020/crab_ZZ_Run2016_sample3
ERROR 2020-07-23 08:19:58.530 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    self.loadLocalCache()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 485, in loadLocalCache
    raise ex
CachefileNotFoundException: Cannot find .requestcache file in CRAB project directory /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/getBTagHist_DeepCSV_07232020/crab_ZZ_Run2016_sample3
INFO 2020-07-23 08:19:58.562 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-23 17:51:21.598 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-23 17:51:21.599 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-23 17:51:21.622 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-23 17:51:21.622 UTC: 	 Executing command: 'status'
DEBUG 2020-07-23 17:51:21.622 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-23 17:51:21.623 UTC: 	 Error: getBTagHist_DeepCSV_07232020/crab_ZTo2Nu_Run2016_sample11 is not a valid CRAB project directory.
ERROR 2020-07-23 17:51:21.624 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07232020/crab_ZTo2Nu_Run2016_sample11 is not a valid CRAB project directory.
INFO 2020-07-23 17:51:21.637 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-23 18:02:06.930 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-23 18:02:06.930 UTC: 	 Running on: Linux login05.hep.wisc.edu 4.4.188-1.el7.elrepo.x86_64 #1 SMP Tue Aug 6 13:44:31 EDT 2019 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-23 18:02:06.968 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-23 18:02:06.968 UTC: 	 Executing command: 'status'
DEBUG 2020-07-23 18:02:06.968 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-23 18:02:06.969 UTC: 	 Error: getBTagHist_DeepCSV_07232020/crab_QCD_Run2016_sample111 is not a valid CRAB project directory.
ERROR 2020-07-23 18:02:06.970 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07232020/crab_QCD_Run2016_sample111 is not a valid CRAB project directory.
INFO 2020-07-23 18:02:06.973 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-25 00:58:26.015 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-25 00:58:26.015 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-25 00:58:26.033 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-25 00:58:26.034 UTC: 	 Executing command: 'status'
DEBUG 2020-07-25 00:58:26.034 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-25 00:58:26.035 UTC: 	 Error: getBTagHist_DeepCSV_07232020/crab_WW_Run2016_sample5 is not a valid CRAB project directory.
ERROR 2020-07-25 00:58:26.036 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07232020/crab_WW_Run2016_sample5 is not a valid CRAB project directory.
INFO 2020-07-25 00:58:26.046 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-26 01:33:42.847 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-26 01:33:42.847 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-26 01:33:42.860 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-26 01:33:42.861 UTC: 	 Executing command: 'status'
DEBUG 2020-07-26 01:33:42.861 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-26 01:33:42.862 UTC: 	 Error: getBTagHist_DeepCSV_07242020/crab_ZTo2L_Run2017_sample111 is not a valid CRAB project directory.
ERROR 2020-07-26 01:33:42.862 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07242020/crab_ZTo2L_Run2017_sample111 is not a valid CRAB project directory.
INFO 2020-07-26 01:33:42.871 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-26 01:43:04.629 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-26 01:43:04.629 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-26 01:43:04.642 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-26 01:43:04.642 UTC: 	 Executing command: 'status'
DEBUG 2020-07-26 01:43:04.642 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-26 01:43:04.643 UTC: 	 Error: getBTagHist_DeepCSV_07242020/crab_TTV_Run2017_sample7 is not a valid CRAB project directory.
ERROR 2020-07-26 01:43:04.643 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07242020/crab_TTV_Run2017_sample7 is not a valid CRAB project directory.
INFO 2020-07-26 01:43:04.645 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-27 16:10:45.216 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-27 16:10:45.216 UTC: 	 Running on: Linux login03.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-27 16:10:45.233 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-27 16:10:45.233 UTC: 	 Executing command: 'status'
DEBUG 2020-07-27 16:10:45.234 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-27 16:10:45.235 UTC: 	 Error: getBTagHist_DeepCSV_07242020/crab_ZZ_Run2017_sample4 is not a valid CRAB project directory.
ERROR 2020-07-27 16:10:45.235 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07242020/crab_ZZ_Run2017_sample4 is not a valid CRAB project directory.
INFO 2020-07-27 16:10:45.245 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-07-28 15:40:36.532 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-07-28 15:40:36.534 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-07-28 15:40:36.570 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-07-28 15:40:36.571 UTC: 	 Executing command: 'status'
DEBUG 2020-07-28 15:40:36.572 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-07-28 15:40:36.856 UTC: 	 Error: getBTagHist_DeepCSV_07242020/crab_ZZ_Run2017_sample is not a valid CRAB project directory.
ERROR 2020-07-28 15:40:36.856 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: getBTagHist_DeepCSV_07242020/crab_ZZ_Run2017_sample is not a valid CRAB project directory.
INFO 2020-07-28 15:40:36.980 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-02 22:59:32.203 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-08-02 22:59:32.203 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-02 22:59:32.236 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-08-02 22:59:32.237 UTC: 	 Executing command: 'status'
DEBUG 2020-08-02 22:59:32.237 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-02 22:59:32.240 UTC: 	 Error: ModuleCommon_07302020/crab_SingleElectron_Run2017_sample7 is not a valid CRAB project directory.
ERROR 2020-08-02 22:59:32.240 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_07302020/crab_SingleElectron_Run2017_sample7 is not a valid CRAB project directory.
INFO 2020-08-02 22:59:32.263 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-07 11:57:46.953 UTC: 	 CRAB Client version: v3.200531
DEBUG 2020-08-07 11:57:46.953 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-07 11:57:46.966 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-08-07 11:57:46.967 UTC: 	 Executing command: 'status'
DEBUG 2020-08-07 11:57:46.967 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-07 11:57:46.968 UTC: 	 Error: ModuleCommon_07302020/crab_TTV_Run2017_sample7 is not a valid CRAB project directory.
ERROR 2020-08-07 11:57:46.968 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200531.00/lib/CRABClient/Commands/SubCommand.py", line 619, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_07302020/crab_TTV_Run2017_sample7 is not a valid CRAB project directory.
INFO 2020-08-07 11:57:46.987 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-12 12:15:51.978 UTC: 	 CRAB Client version: v3.200719
DEBUG 2020-08-12 12:15:51.979 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-12 12:15:52.014 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-08-12 12:15:52.015 UTC: 	 Executing command: 'status'
DEBUG 2020-08-12 12:15:52.015 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-12 12:15:52.018 UTC: 	 Error: ModuleCommon_08092020/crab_ZTo2Nu_Run2018_sample7 is not a valid CRAB project directory.
ERROR 2020-08-12 12:15:52.018 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/SubCommand.py", line 631, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08092020/crab_ZTo2Nu_Run2018_sample7 is not a valid CRAB project directory.
INFO 2020-08-12 12:15:52.061 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-12 12:18:04.849 UTC: 	 CRAB Client version: v3.200719
DEBUG 2020-08-12 12:18:04.850 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-12 12:18:04.872 UTC: 	 OpenSSl version: OpenSSL 1.0.2k-fips  26 Jan 2017 (Library: OpenSSL 1.0.2d-fips 9 Jul 2015)
DEBUG 2020-08-12 12:18:04.872 UTC: 	 Executing command: 'status'
DEBUG 2020-08-12 12:18:04.873 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-12 12:18:04.875 UTC: 	 Error: ModuleCommon_08092020/crab_WZ_Run2018_sample5 is not a valid CRAB project directory.
ERROR 2020-08-12 12:18:04.876 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/SubCommand.py", line 278, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200719.00/lib/CRABClient/Commands/SubCommand.py", line 631, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08092020/crab_WZ_Run2018_sample5 is not a valid CRAB project directory.
INFO 2020-08-12 12:18:04.882 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-27 04:05:31.863 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-08-27 04:05:31.864 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-27 04:05:31.864 UTC: 	 Executing command: 'status'
DEBUG 2020-08-27 04:05:31.864 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-27 04:05:31.865 UTC: 	 Error: ModuleCommon_08252020/crab_ZTo2L_Run2017_sample12 is not a valid CRAB project directory.
ERROR 2020-08-27 04:05:31.865 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08252020/crab_ZTo2L_Run2017_sample12 is not a valid CRAB project directory.
INFO 2020-08-27 04:05:31.903 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-08-30 22:32:01.860 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-08-30 22:32:01.860 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-08-30 22:32:01.861 UTC: 	 Executing command: 'status'
DEBUG 2020-08-30 22:32:01.861 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-08-30 22:32:01.862 UTC: 	 Error: ModuleCommon_08282020/crab_TTV_Run2018_sample5 is not a valid CRAB project directory.
ERROR 2020-08-30 22:32:01.862 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_TTV_Run2018_sample5 is not a valid CRAB project directory.
INFO 2020-08-30 22:32:01.897 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-04 05:26:03.535 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-04 05:26:03.535 UTC: 	 Running on: Linux login04.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-04 05:26:03.535 UTC: 	 Executing command: 'status'
DEBUG 2020-09-04 05:26:03.536 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-04 05:26:03.536 UTC: 	 Error: ModuleCommon_08282020v2/crab_ttbarPlusJets_Run2018_sample0 is not a valid CRAB project directory.
ERROR 2020-09-04 05:26:03.537 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020v2/crab_ttbarPlusJets_Run2018_sample0 is not a valid CRAB project directory.
INFO 2020-09-04 05:26:03.562 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-05 03:35:37.567 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-05 03:35:37.568 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-05 03:35:37.568 UTC: 	 Executing command: 'status'
DEBUG 2020-09-05 03:35:37.568 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-05 03:35:37.569 UTC: 	 Error: ModuleCommon_08282020/crab_singleTop_Run2018_sample2] is not a valid CRAB project directory.
ERROR 2020-09-05 03:35:37.569 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_singleTop_Run2018_sample2] is not a valid CRAB project directory.
INFO 2020-09-05 03:35:37.578 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-05 03:43:24.002 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-05 03:43:24.002 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-05 03:43:24.002 UTC: 	 Executing command: 'status'
DEBUG 2020-09-05 03:43:24.003 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-05 03:43:24.004 UTC: 	 Error: ModuleCommon_08282020/crab_singleTop_Run2018_sample5 is not a valid CRAB project directory.
ERROR 2020-09-05 03:43:24.004 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_singleTop_Run2018_sample5 is not a valid CRAB project directory.
INFO 2020-09-05 03:43:24.006 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-05 07:44:15.388 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-05 07:44:15.388 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-05 07:44:15.388 UTC: 	 Executing command: 'status'
DEBUG 2020-09-05 07:44:15.389 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-05 07:44:15.389 UTC: 	 Error: ModuleCommon_08282020/crab_ZTo2L_Run2018_sample111 is not a valid CRAB project directory.
ERROR 2020-09-05 07:44:15.390 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_ZTo2L_Run2018_sample111 is not a valid CRAB project directory.
INFO 2020-09-05 07:44:15.392 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-05 09:10:51.545 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-05 09:10:51.545 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-05 09:10:51.545 UTC: 	 Executing command: 'status'
DEBUG 2020-09-05 09:10:51.546 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-05 09:10:51.547 UTC: 	 Error: ModuleCommon_08282020/crab_WW_Run2018_sample3 is not a valid CRAB project directory.
ERROR 2020-09-05 09:10:51.547 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_WW_Run2018_sample3 is not a valid CRAB project directory.
INFO 2020-09-05 09:10:51.549 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-05 09:23:31.474 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-05 09:23:31.475 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-05 09:23:31.475 UTC: 	 Executing command: 'status'
DEBUG 2020-09-05 09:23:31.475 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-05 09:23:31.476 UTC: 	 Error: ModuleCommon_08282020/crab_TTV_Run2018_sample5 is not a valid CRAB project directory.
ERROR 2020-09-05 09:23:31.476 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_TTV_Run2018_sample5 is not a valid CRAB project directory.
INFO 2020-09-05 09:23:31.479 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-06 14:35:34.690 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-06 14:35:34.690 UTC: 	 Running on: Linux login01.hep.wisc.edu 4.4.224-1.el7.elrepo.x86_64 #1 SMP Wed May 20 09:26:31 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-06 14:35:34.690 UTC: 	 Executing command: 'status'
DEBUG 2020-09-06 14:35:34.690 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-06 14:35:34.691 UTC: 	 Error: ModuleCommon_08282020/crab_Single0 is not a valid CRAB project directory.
ERROR 2020-09-06 14:35:34.692 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_08282020/crab_Single0 is not a valid CRAB project directory.
INFO 2020-09-06 14:35:34.694 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-15 16:47:14.072 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-15 16:47:14.072 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-15 16:47:14.072 UTC: 	 Executing command: 'status'
DEBUG 2020-09-15 16:47:14.072 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-15 16:47:14.073 UTC: 	 Error: ModuleCommon_09142020/crab_WZ_Run2016_sample6 is not a valid CRAB project directory.
ERROR 2020-09-15 16:47:14.074 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_09142020/crab_WZ_Run2016_sample6 is not a valid CRAB project directory.
INFO 2020-09-15 16:47:14.098 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
DEBUG 2020-09-15 16:47:28.328 UTC: 	 CRAB Client version: v3.200812
DEBUG 2020-09-15 16:47:28.328 UTC: 	 Running on: Linux login06.hep.wisc.edu 4.4.216-1.el7.elrepo.x86_64 #1 SMP Wed Mar 11 09:13:43 EDT 2020 x86_64 x86_64 x86_64 GNU/Linux - CentOS Linux 7 (Core)
DEBUG 2020-09-15 16:47:28.328 UTC: 	 Executing command: 'status'
DEBUG 2020-09-15 16:47:28.329 UTC: 	 Found CRAB cache file /afs/hep.wisc.edu/home/vshang/.crab3
ERROR 2020-09-15 16:47:28.330 UTC: 	 Error: ModuleCommon_09142020/crab_WZ_Run2016_sample7 is not a valid CRAB project directory.
ERROR 2020-09-15 16:47:28.330 UTC: 	 Caught ClientException exception
Traceback (most recent call last):
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 165, in <module>
    client()
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/bin/crab", line 150, in __call__
    self.cmd = sub_cmd(self.logger, args[1:])
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/status.py", line 41, in __init__
    SubCommand.__init__(self, logger, cmdargs)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 270, in __init__
    SubCommand.validateOptions(self)
  File "/cvmfs/cms.cern.ch/share/cms/crab-prod/v3.200812.00/lib/CRABClient/Commands/SubCommand.py", line 623, in validateOptions
    raise ConfigurationException(msg)
ConfigurationException: [91mError[0m: ModuleCommon_09142020/crab_WZ_Run2016_sample7 is not a valid CRAB project directory.
INFO 2020-09-15 16:47:28.339 UTC: 	 Log file is /afs/hep.wisc.edu/user/vshang/public/tDM_nanoAOD/CMSSW_10_2_9/src/PhysicsTools/NanoAODTools/crab/crab.log
